controllers/storagecluster/storagecluster_controller.go

```diff
@@ -91,6 +91,10 @@ type StorageClusterReconciler struct {
 	IsNoobaaStandalone bool
 }
 
+func skipNoobaaCrdWatch() bool {
```

Comment:
No need to make this a function, just use it inline.

Reply:
removed it